.github/workflows/build.yml

```diff
@@ -1,25 +1,49 @@
 name: build
 
 on:
-  # Trigger analysis when pushing in main or pull requests, and when creating
-  # a pull request. 
   push:
+    paths:
+      # specific folder locations
+      - "deeprankcore/**"
```

Comment:
Instead of listing all paths, does it work if we list only the ones we want to exclude? If yes, I'd go for that. Less verbose and easier to maintain. Same for the others yml

Reply:
why is that less verbose? I think there's many more file types and locations to exclude than there are to include.
I can try to find out if it's possible to maintain a single list of locations to include so we can refer to it in each yml file rather than listing them each time (although I don't really want to spend a lot of extra time on this).

Reply:
I guess ignore is safer, cause it's less likely we list something there that shouldn't be ignored as opposed to forgetting to add something to the current list that should not be ignored.